lib/Monads/nondet/Nondet_Reader_Option.thy

```diff
@@ -70,6 +70,10 @@ lemma gets_the_obind:
   "gets_the (f |>> g) = gets_the f >>= (\<lambda>x. gets_the (g x))"
   by (rule ext) (simp add: monad_simps obind_def split: option.splits)
 
+lemma gets_the_Some_get:
```

Comment:
should we mark this one as `[simp]`? I can't imagine ever wanting `gets_the Some`

Reply:
Yeah that sounds good to me, if others are happy with it.

Reply:
yes, let's